common/levy_templates/01_anatolian.txt

```diff
@@ -8,16 +8,17 @@
 levy_anatolian = {	#General Anatolian
 	default = no
 
-	light_infantry = 0.6
-	light_cavalry = 0.3
+	light_infantry = 0.8
+	light_cavalry = 0.1
 	heavy_cavalry = 0.1
 }
 
 levy_armenian = {	#Armenians
 	default = no
 
 	light_infantry = 0.7
-	horse_archers = 0.2
+	archers = 0.1
```

Comment:
looks like they have 4 units in their levy and over 100%

Reply:
Isn't that from the previous culture levy ?